app/views/catalog/availability_modal.html.erb

```diff
@@ -14,7 +14,9 @@
         <input type="search" class="form-control border-start-0" placeholder="Search for a copy, volume, or issue" data-action="availability-search#search" id="availability-search-input" />
       </div>
       <%= render Searchworks4::OnlineAccordionComponent.new(document: @document, toggled_library: params[:library]) %>
-      <%= render Searchworks4::AvailabilityAccordionComponent.with_collection(@document.holdings.libraries.compact_blank, document: @document, toggled_library: params[:library]) %>
+      <%= render Searchworks4::AvailabilityAccordionComponent.with_collection(@document.holdings.libraries.reject { |library| library.code.start_with?('SAL') }.compact_blank, document: @document, toggled_library: params[:library])%>
+      <% sal_libraries = @document.holdings.libraries.select { |library| library.code.start_with?('SAL')} %>
```

Comment:
Can you put this as a method on `SolrDocument`?  I'd prefer not to see details about the model in views.